index.js

```diff
@@ -100,24 +130,46 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+
+function triple(number) {
+  const tripleResult = (number * 3)
+  return tripleResult
+}
+
+function myMap(array, callback) {
+  const result = []
+  for (let i = 0; i < array.length; i++) {
+    const element = array[i]
+    const value = callback(element)
+    result.push(value)
+  }
+  return result
+}
+
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
+
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
+
+function greet(name) {
+  const namesArray = `Hi, ${name}!`
```

Comment:
While variable names technically don't matter, think about the data type for `namesArray`